pages/login.vue

```diff
@@ -67,6 +64,13 @@ export default Vue.extend({
       disclaimer: ''
     };
   },
+  async fetch() {
```

Comment:
Same here, it'd make more sense to check this before the page is rendered, so in `asyncData`.